src/debug/args-test.js

```diff
@@ -87,7 +87,7 @@ class ArgsTest {
         const breakpointFile = path.join(engine.cwd, this.sourcePath, 'ArgsTest.java');
         const expectedLine = 20;
         const outputList = [];
-        var assertCount=0;
+        let assertCount=0;
```

Comment:
add space to the left and right of = operator.

Reply:
https://github.com/elierotenberg/coding-styles/blob/master/es6.md 
`// bad
const a = b+c;
const a= b + c;

// good
const a = b + c;`

Reply:
OK, I will be care full with this and update the code